From a352d973e7901db8d46322e1cb7fdae5bf920967 Mon Sep 17 00:00:00 2001
From: Radostin Stoyanov <radostin@redhat.com>
Date: Fri, 10 Dec 2021 15:35:29 +0000
Subject: [PATCH 064/249] ci: install procps in Alpine

The version of ps in Alpine image by default is very limited.
It is based on the one from busybox and doesn't support options
such as '-p'.

Signed-off-by: Radostin Stoyanov <radostin@redhat.com>
---
 scripts/build/Dockerfile.alpine | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/build/Dockerfile.alpine b/scripts/build/Dockerfile.alpine
index a6579c0bb..cab72e8a1 100644
--- a/scripts/build/Dockerfile.alpine
+++ b/scripts/build/Dockerfile.alpine
@@ -6,6 +6,7 @@ RUN apk update && apk add \
 	bash \
 	build-base \
 	coreutils \
+	procps \
 	git \
 	gnutls-dev \
 	libaio-dev \
-- 
2.35.1

